client/src/pages/Apply/Apply.js

```diff
@@ -447,59 +450,55 @@ const Apply = ({ navigation }) => {
                         fontFamily="heading"
                         onPress={() => Linking.openURL('https://www.7000.org/about-3-1')}
                       >
-                        I agree to the{' '}
-                        <Text
-                          fontFamily="heading"
-                          onPress={() => Linking.openURL('https://www.7000.org/about-3-1')}
-                        >
-                          Terms and Conditions
-                        </Text>
+                        Terms and Conditions
                       </Text>
                     </Text>
                   </View>
                 </Checkbox>
               </View>
 
+              {/* Weird styling to accomodate native-base checkbox text centering */}
               <View style={styles.checkboxes}>
                 <Checkbox
+                  style={{ display: 'flex', alignSelf: 'flex-end' }}
                   value="accepted"
                   colorScheme="danger"
                   onChange={setAcceptTerms}
+                  aria-label="asdf"
+                />
+                <Text
+                  fontFamily="body"
+                  fontWeight="regular"
+                  color="black"
+                  fontStyle="normal"
+                  fontSize="md"
+                  style={{ paddingLeft: 8, top: -3 }}
                 >
-                  <View>
-                    <Text
-                      fontFamily="body"
-                      fontWeight="regular"
-                      color="black"
-                      fontStyle="normal"
-                      fontSize="md"
-                    >
-                      I would like a team member from 7000 Languages to follow
-                      up with me about creating additional resources for my
-                      language.
-                    </Text>
-                  </View>
-                </Checkbox>
+                  I would like a team member from 7000 Languages to follow up
+                  with me about creating additional resources for my language.
+                </Text>
               </View>
             </View>
+            <Box style={styles.termsText}>
+              <StyledButton
+                title="Submit"
+                variant="primary"
+                onPress={onSubmit}
+              />
+              <Text
+                fontFamily="body"
+                fontWeight="regular"
+                color="gray.medium"
+                fontStyle="normal"
+                fontSize="sm"
+              >
```

Comment:
Can you move this text (By selecting this button, you confirm...) above the button to match the Figma designs?

Reply:
done